mesa/experimental/observer.py

```diff
@@ -0,0 +1,77 @@
+from collections import defaultdict
+
+import pandas as pd
+
+
+class DataCollector:
+    """
+    Example: a model consisting of a hybrid of Boltzmann wealth model and
```

Comment:
aka what if during the "we are the 99%" protest, people are constantly gifting money randomly, to the point that there are emergent 1% within the protesters.